src/main/java/frc/robot/subsystems/elevator/ElevatorSubsystem.java

```diff
@@ -0,0 +1,38 @@
+package frc.robot.subsystems.elevator;
+
+import edu.wpi.first.wpilibj.RobotBase;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.Commands;
+
+public class ElevatorSubsystem {
+  private ElevatorIO elevatorIO;
+
+  public ElevatorSubsystem() {
+    if (RobotBase.isSimulation()) {
+      elevatorIO = new ElevatorIOReal();
+    }
```

Comment:
need an else here.  If you make the field `final` the compiler would tell you the mistake.